celsos_bookSearch/celsos_bookSearch/Model/SearchResult.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  SearchResult.swift
+//  celsos_bookSearch
+//
+//  Created by Celso Junio Simões de Oliveira Santos on 21/05/21.
+//
+
+import Foundation
+
+public struct SearchResult: Decodable, Encodable{
+    var resultCount:Int?
```

Comment:
You can do the same update here as suggested by Flavia
`var resultCount: Int?`

Reply:
Done